### ISSUE TITLE
Bad wheel. 

### ISSUE THREAD

**Issue by Carreau** [getnikola/nikola#2102]
The wheel is wrong and have file that should not be there , 

In particular a `/ipynb/__init__.pyc` that prevent from loading the `ipynb.py` of the same folder. 

I'm not even sure how this could have happend. 
````bash
$ # unzip wheel, cd inside...
$ pwd; tree

.../Nikola-7.7.1-py2.py3-none-any/nikola/plugins/compile
.
├── __init__.py
├── __init__.pyc
.... // snip
├── ipynb
│   └── __init__.pyc  ##wrong
├── ipynb.plugin
├── ipynb.py
├── ipynb.pyc
├── markdown
│   ├── __init__.py
...// snip
│   ├── youtube.py
│   └── youtube.pyc
└── rest.plugin

6 directories, 85 files

```

**Comment by Kwpolska** [comment]
This was probably a leftover from the days when ipynb was a subpackage.  I uploaded a fixed `7.7.1.1` wheel and added a global exclude rule to prevent this from happening again.

@ralsina, please remove all .pyc files from your local tree and look for leftover packages.  In particular, n/p/command/auto.pyc and n/p/compile/ipynb/* need your attention.

**Comment by Carreau** [comment]
Great thanks !
